apps/root/src/services/earnService.ts

```diff
@@ -286,6 +286,7 @@ export class EarnService extends EventsManager<EarnServiceData> {
         ...strategy,
         lastUpdatedAt: nowInSeconds(),
         userPositions: includedUserStrategies,
+        hasFetchedHistoricalData: 'hasFetchedHistoricalData' in strategy ? strategy.hasFetchedHistoricalData : false,
```

Comment:
is this ternary necessary?

Reply:
For typescript yes, since this receives either an SdkStrategy or a SavedSdkStrategy